### PR TITLE
Refactor hardcoded JS CDN links to _protected variables

### PR DESCRIPTION
**Pull request by and-viceversa**
`folium.py` exposes `_default_js` and `_default_css` at a high level. This is good, because you can override the CDN URL to point elsewhere. In my case, to point to a static assets folder.

The plugins do not do this. This PR moves the URLs out of their respective `render()` methods to expose them in a similar way. This shouldn't change any existing behavior.

Also, 98% of the changes are fixing `flake8` results. Sorry about that but I went by the contributing guideline.

### CODE REVIEW

**Comment by Conengmo** [comment]
Hi @and-viceversa, thanks for your PR, I think this is a good idea!

For anyone reading along, this was discussed in #351 and more recently in #1312.

Two things:

I'd prefer if each module had the same variable name for the JS and css links. Since we already have `_default_js` and `_default_css` in `folium.py` we can use those names.

Can you remove the style changes in code you otherwise wouldn't touch? It makes reviewing a lot more work, and since I'm doing this in my free time that's something to look out for. I'll update the contributing guide to reflect this as well.

**Comment by and-viceversa** [comment]
Please see updated PR.

The plugins that use them now follow the `_default_js` and `_default_css` pattern.

The `render()` methods use the same `for` loop as `folium.py`.

Most plugins only use 1 JS or CSS file, so this is a bit overkill. But now it's uniform.

Thanks for your quick feedback. Let me know what else I can do on this.

**Comment by Conengmo** [comment]
Looks good @and-viceversa ! I'll check in more detail before merging, but at this time there's nothing else for you to do. Don't worry about the commits, we squash and merge.

File: folium/plugins/timestamped_geo_json.py [hunk 1]
```diff
@@ -9,6 +9,29 @@
...
+]
+
+_default_css = [
+    ('highlight.js_css',
+     'https://cdnjs.cloudflare.com/ajax/libs/highlight.js/8.4/styles/default.min.css'),
+    ('leaflet.timedimension_css',
+     'https://rawcdn.githack.com/socib/Leaflet.TimeDimension/master/dist/leaflet.timedimension.control.min.css'),
+    # noqa
+    ('bootstrap_theme_css',
+     'https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap-theme.min.css')
```
Comment (Conengmo) [inline folium/plugins/timestamped_geo_json.py:32]: I don't see this one in the plugin. It's in `folium.py` though. Maybe a copy-paste error?

Comment (and-viceversa) [reply folium/plugins/timestamped_geo_json.py]: Yeah looks like simple oversight on my part. Deleted Bootstrap CSS theme.

**Comment by Conengmo** [comment]
Thanks for your work on this @and-viceversa !

**Pull request merged**